cartridge/argparse.lua

```diff
@@ -194,6 +194,7 @@ local box_opts = {
     election_mode            = 'string', -- **string**
     election_timeout         = 'number', -- **number**
     election_fencing_enabled = 'boolean', -- **boolean**
+    testing_number_boolean   = 'number|boolean', -- **number|boolean**
```

Comment:
No, please don't mix box options with some test values.

AFAIK there is a way to add some custom options to argparse without doing that. Or you can export this table somehow and modify in your test.